Gopkg.lock

```diff
@@ -114,6 +114,17 @@
   revision = "b848ebf618ee2a8131b88126469b693aa035457f"
   version = "1.1.2"
 
+[[projects]]
+  digest = "1:28c998a6447b4d28ca5e0b2d045bf00affe03e3494019a583eee2803472be09c"
+  name = "github.com/asecurityteam/transportd"
```

Comment:
Something is off here. This adds the project as a dependency of itself. That shouldn't be the case. I think this would happen if the project was in GOPATH but not at the right location. Do you have the repo cloned at `${GOPATH}/src/github.com/asecurityteam/transportd`?

Reply:
oooohhh, I see, I think I made my own dev directory. I will fix that and see.

Reply:
that fixed it.